pom.xml

```diff
@@ -167,7 +167,7 @@
       <dependency>
         <groupId>org.apache.logging.log4j</groupId>
         <artifactId>log4j-1.2-api</artifactId>
-        <version>2.17.2</version>
+        <version>2.19.0</version>
         <scope>provided</scope>
       </dependency>
```

Comment:
For log4j, it ships a bom, that can be used to manage dependency versions for all log4j artifacts. Actually, I'm wondering why this project has a dependency on the log4j 1.2 API at all. It probably doesn't need this, unless we're just trying to converge transitive dependency versions.